DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc

```diff
@@ -39,7 +39,7 @@ int main() {
 
     // Inner scope to deallocate memory before destroying the stream.
     {
-      TracksSoACollection<pixelTopology::Phase1> tracks_d(queue);
+      TracksSoACollection tracks_d({{1000, 5000}}, queue);
```

Comment:
I assume that
```suggestion
      TracksSoACollection tracks_d({1000, 5000}, queue);
```
does not work ?

Reply:
Yep, it doesn't

```bash
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
   42 |       TracksSoACollection tracks_d({1000, 5000}, queue);
      |                                                       ^
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc: In function 'int main()':
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
   42 |       TracksSoACollection tracks_d({1000, 5000}, queue);
      |                                                       ^
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc: In function 'int main()':
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
   42 |       TracksSoACollection tracks_d({1000, 5000}, queue);
      |                                                       ^
src/DataFormats/TrackSoA/test/alpaka/TrajectoryStateSoA_t.cc:42:55: error: missing braces around initializer for 'std::__array_traits<int, 2>::_Type' {aka 'int [2]'} [-Werror=missing-braces]
```